Chrysalis-Papilio/Soen490-Papilio-Website
# PAPILIO-36 Delete activity completed

## General Information


**Issue number**: Papilio-36


**Task description:**
Delete activity was added. Business users can now view, add and delete an activity on the website.

## Manual Testing
To test the validation follow these steps:
1. start the backend
`npm run build && npm start`
2. start the website
`npm start`
3. On the main page navigate to the login page
4. Insert the admin account information
5. Once logged in, navigate to the "Activity Manager" tab
6. You should see all your activities
7. Click on the "Delete activity" button
8. Checkboxes should appear for you to select the activities you would like to delete
9. Click on "Delete activity" button at the bottom to delete the selected activities
10. Activities deleted should not be visible anymore


## Testing 
To run all unit tests, run the following code: `npm test`
To run the coverage, run the following code: `npm run coverage`
